pkgs/top-level/aliases.nix

```diff
@@ -127,6 +127,7 @@ mapAliases {
   archiveopteryx = throw "archiveopteryx depended on an unsupported version of OpenSSL and was unmaintained"; # Added 2024-01-03
   ardour_6 = throw "ardour_6 has been removed in favor of newer versions"; # Added 2023-10-13
   aria = aria2; # Added 2024-03-26
+  armcord = legcord; # Added 2024-10-11
```

Comment:
Given the legal threats mentioned in the release notes, maybe we should make this a `throw` with an explanation, just in case.

Reply:
No qualms with going that route, especially given the upstream expectation is an uninstall/reinstall.

Reply:
@emilazy If the throw looks good, I will work on making the backport for this as I am not 100% sure the auto-backport will function (the build for 24.05 goes from packages instead of source, so the file is very different). 

Do we also need to remove the package from older than 24.05 given the legal aspect?

Reply:
I think we can wait for Arm to complain for that. We don’t have a good way of making such mass changes or purging stuff from the binary cache unless absolutely necessary. Upstream haven’t removed their old releases either; this PR should show enough good‐faith compliance.